<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKalkulacijaListaClass</class>
 <widget class="QMyBaseFormWidget" name="QKalkulacijaListaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2400</width>
    <height>2400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QKalkulacijaLista</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1056</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0" colspan="4">
     <widget class="QTableView" name="tableView_2">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#0000ff;&quot;&gt;Приемници - Калкулации - Листа на добавувачи&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="4">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>  Корегирај постоечка</string>
        </property>
        <property name="icon">
         <iconset resource="sterna.qrc">
          <normaloff>:/sterna/Resources/key_f3.png</normaloff>:/sterna/Resources/key_f3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Внеси нова</string>
        </property>
        <property name="icon">
         <iconset resource="sterna.qrc">
          <normaloff>:/sterna/Resources/key_f2.png</normaloff>:/sterna/Resources/key_f2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Избришија ја постоечката</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Copy Table in Clipboard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2" colspan="4">
     <widget class="QTableView" name="tableView">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="4">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Вк. Набавен без ДДВ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Вк. влезен ДДВ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Вк. Набавен со ДДВ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Вк. маржа</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Вк. продажен без ДДВ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Вк. пресметан ДДВ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Вк. продажен со ДДВ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMyBaseFormWidget</class>
   <extends>QWidget</extends>
   <header>qmybaseformwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="sterna.qrc"/>
 </resources>
 <connections/>
</ui>
